### ISSUE TITLE
Parity is connecting very few peers or no peer at all while netPeers showing 

### ISSUE THREAD

**Issue by kerematam** [paritytech/parity#9109]
_Before filing a new issue, please **provide the following information**._

> I'm running:
>
> - **Which Parity version?**: Parity/v1.10.8-stable-bd20c45-20180628/x86_64-linux-gnu/rustc1.27.
> - **Which operating system?**: Linux
> - **How installed?**: Docker
> - **Are you fully synchronized?**: yes
> - **Which network are you connected to?**: ethereum
> - **Did you try to restart the node?**: yes

I've tried to remove nodes.json and restart parity as it is suggested previously in few issues. But no hope for my case. I have very similar setup for Robsten network on different PC and different network and i have far more connected client. My transactions is not broadcasted well enough because of this issue.

Output of `parity_netPeers` rpc method brinds me 
1 connected 
0 active 
46 peers
I can share more detailed logs if needed.
```
$ curl --data '{"method":"parity_netPeers","params":[],"id":1,"jsonrpc":"2.0"}' -H "Content-Type: application/json" -X POST http://localhost | jq '.result | .connected , .active, .peers | length'
1
0
46
```

**Comment by Tbaut** [comment]
Some improvements have been made on 1.11 (will be marked as stable in the next days).
Could you please give it a try? How long did you wait until having this amount of peers?

**Comment by kerematam** [comment]
Yeah. Ropsten :D. Well, i waited more than one day. I will be trying 1.11. Hopefully it will solve my problem.
